Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -510,7 +506,7 @@ import Foundation
     func fetchAPITiming(path: String, startTime: Int, endTime: Int) {
         let cleanedPath = path.replacingOccurrences(of: "/merchants/([A-Za-z0-9]+)/client_api", with: "", options: .regularExpression)
 
-        if (shouldSendAPIRequestLatency || path.contains("v1/configuration")) && cleanedPath != "/v1/tracking/batch/events" {
+        if cleanedPath != "/v1/tracking/batch/events" {
```

Comment:
It looks like we aren't currently sending anything for requests going through GraphQL (for example `Card` flows). What do we want to send in these instances? cc: @scannillo

Reply:
Ah, that's a great catch! (Yet another reason to move to 1 URLSession implementation 🐔)

We should have the same metrics for GraphQL calls too, but since "endpoint" is always the same, we'd probably need to log the mutation name instead. I'm OK if we want to make a separate ticket to tackle that.

Reply:
with this change [5783145](https://github.com/braintree/braintree_ios/pull/1321/commits/5783145b461ebb16ab8703587856ac903f4e2fe3) we are able to send this event for GraphQL request, yeap as Sammy said, the path will always the same

Reply:
I'd prefer to address GraphQL mutation in a separate ticket/PR, but I'm open to whatever you prefer

Reply:
I'm find tackling this in a separate ticket. Created DTMOBILES-683 for this.

Reply:
Awesome, thanks for creating that ticket